Cargo.toml

```diff
@@ -59,7 +59,15 @@ assets = [{ source = "target/release/topgrade", dest = "/usr/bin/topgrade" }]
 git = "*"
 
 [package.metadata.deb]
+name = "topgrade"
+maintainer = "Topgrade Maintainer <maintainer@noreply.local>"
+copyright = "2024, Topgrade Team"
+license-file = ["LICENSE", "0"]
```

Comment:
What is this `"0"`?

Reply:
`license-file` asks for both a path to a filename with the license information, and an integer indicating the number of lines to strip from the top of it when adding it to the package (for files that have non-license-related preambles at the start).  Specifying `"0"` means to just include the license file exactly as is.

Reply:
Thanks for letting me know!